tests/e2e/cucumber/features/oidc/refreshToken.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Token renewal using refresh token
+
+  As a user
+  I want the web application to automatically refresh its access token using a refresh token,
+  So that I can be confident the application will not encounter issues related to expired access tokens.
+
+
+  Scenario: access token renewal via refresh token
+    Given "Admin" creates following users using API
+      | id    |
+      | Alice |
+    And "Admin" assigns following roles to the users using API
+      | id    | role        |
+      | Alice | Space Admin |
+    And "Alice" logs in
+    And "Alice" opens the "files" app
+    And "Alice" navigates to the projects space page
+    And "Alice" creates the following project spaces
+      | name | id     |
+      | team | team.1 |
+    When "Alice" waits for token renewal via refresh token
+    And "Alice" navigates to the project space "team.1"
+    And "Alice" creates the following resources
+      | resource     | type   |
+      | space-folder | folder |
+    Then following resources should be displayed in the files list for user "Alice"
+      | resource     |
+      | space-folder |
+    When "Alice" navigates to new tab
+    And "Alice" waits for token to expire
+    And "Alice" closes the current tab
+    And "Alice" opens the "files" app
+    And "Alice" creates the following resources
+      | resource          | type    | content   |
+      | PARENT/parent.txt | txtFile | some text |
```

Comment:
>On top of that, for the second scenario (with offline_access / refresh token) it would be great to have an additional test that navigates away from web entirely and comes back after the access token has expired to validate that a new access token is obtained

and for above statement. we have added this scenarios steps.

Reply:
```
    And "Alice" waits for token to expire
    And "Alice" closes the current tab
```

Shouldn't this be the other way round?

Reply:
>test that navigates away from web entirely and comes back after the access token has expired to validate that a new access token is obtained

I tried to implement this
- navigates away from web (go to new tab)
- wait for token to expire
- comes back (to web) to validate that a new access token is obtained